### PR TITLE
Updated the README.md and the configs/README.md

### PR DESCRIPTION
**Pull request by milutinke**
Since everything has been moved to the documentation website, I've updated the README.mds to make them cleaner looking and to point at important stuff in the documentation.

### CODE REVIEW

**Comment by milutinke** [comment]
@ORelio Can I merge this if it is fine by you?
Since there is a lot of people who do not know about the documentation and are trying to use mono and opening issues.
I've moved everything and I'm working on a draft of phase 2.

File: MinecraftClient/config/README.md [hunk 1]
```diff
@@ -1,517 +1,4 @@
...
-If you still have any question after reading this file, you can get support here:
-
- - GitHub Issues: https://github.com/MCCTeam/Minecraft-Console-Client/issues (You can contact the MCC Team here)
- - Minecraft Forums: http://www.minecraftforum.net/topic/1314800-/ (Thead not maintained by the MCC Team anymore)
-
-Code contributions, bug reports and any kind of comments are also highly appreciated :)
-
-_© 2012-2020 ORelio_
-_© 2020-2021 The MCC Team_
+This section has been moved to appropriate sections on our new [Documentation website](https://mccteam.github.io/docs/).
```
Comment (ORelio) [inline MinecraftClient/config/README.md:4]:
```suggestion
This page has been moved to appropriate sections on our new [Documentation website](https://mccteam.github.io/docs/).
```

File: README.md [hunk 2]
```diff
@@ -1,19 +1,31 @@
-Minecraft Console Client
-========================
+<div align="center">
+
+<img src="https://i.pics.rs/LLDhE.png" alt="Logo"/>
+
+# Minecraft Console Client (MCC)
+
+[Documentation](https://mccteam.github.io/docs/) | [Download](#download-🔽) | [Installation](https://mccteam.github.io/docs/guide/installation.html) | [Configuration](https://mccteam.github.io/docs/guide/configuration.html) | [Usage](https://mccteam.github.io/docs/guide/usage.html)
```
Comment (ORelio) [inline README.md:7]: The Download link does not seems to jump to the appropriate section, likely because of the 🔽 emoticon 🤔

**Review by ORelio — APPROVED** [review]
I'm totally fine with the changes. Just made a small remarks by the way:

File: README.md (hunk 2 above)
Comment (milutinke) [reply README.md:7]: I'll remove it then.

**Pull request merged**